lib/csp.ts

```diff
@@ -39,9 +39,6 @@ export const data = {
         'localhost:*',
```

Comment:
All else is safe - but I'd appreciate an extra pair of eyes on the security policy file. Probably best test in staging too.

Reply:
@AbrilRBS  does this look right to you?

Reply:
I'm not 100% sure we're even using this honestly (but defer to Abril 100% on this)